app/controllers/serviceLearning/routes.py

```diff
@@ -27,38 +28,72 @@ def serviceCourseManagement(username=None):
         flash("Unauthorized to view page", 'warning')
         return redirect(url_for('main.events'))
 
+@serviceLearning_bp.route('/serviceLearning/editProposal/<courseID>', methods=['GET', 'POST'])
+def slcEditProposal(courseID):
+    questionData = CourseQuestion.select().where(CourseQuestion.course == courseID)
+    questionanswers = [question.questionContent for question in questionData]
+    courseData = questionData[0]
+    courseInstructor = CourseInstructor.select().where(CourseInstructor.course == courseID)
+
+    isRegularlyOccuring = ""
+    isAllSectionsServiceLearning = ""
+    isPermanentlyDesignated = ""
+
+    if courseData.course.isRegularlyOccuring:
+        isRegularlyOccuring = "checked"
+    if courseData.course.isAllSectionsServiceLearning:
+        isAllSectionsServiceLearning = "checked"
+    if courseData.course.isPermanentlyDesignated:
+        isPermanentlyDesignated = "checked"
```

Comment:
We use the "Checked" keyword because we pass it to the font end using jinja and if it says "Checked" that is a keyword the HTML uses to check the box if it is not "Checked" HTML will ignore it. We could make it True or False and then uses jinja logic to make it checked or not but I figured it would be better to do the logic on the Python side.

Reply:
What if you changed it to not be a checkbox? Then "checked" no longer makes sense, and you're having to change python code for a UI concern. Let jinja (the View) handle UI decisions.